index.html

```diff
@@ -0,0 +1,219 @@
+                    <h2 class="let-connect" id="lets-connect_title">LET'S CONNECT</h2>
+                </div>
+                <ul class="contact_us_wrapper">
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a></li>
+                    <li><a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+                    <li><a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-medium-m"></i></a></li>  
+                </ul>
+                <button class="see-project" id="project_about">See Project</button>
+            </div>
+            <div class="performance__wrapper">
+                <div class="performance">
+                    <h3 class="languages-title">Languages</h3>
+                    <ul class="language-details">
+                        <li class="javascript"><img class="platform_img" src="./img/javascript.png" alt="javscript picture"><a>Jav</a></li>
+                        <li class="html"><img class="platform_img" src="./img/html.png" alt="html picture">HTML</li>
+                        <li class="css"><img class="platform_img" src="./img/css.png" alt="css picture">CSS</li>
+                    </ul>
+                    <div class="total__wrapper">
+                        <div class="platform__wrapper">
+                            <h3 class="frameworks-title">Frameworks</h3>
+                            <ul class="bar__wrapper">
+                                <li class="bar"></li>
+                                <li class="bar"></li>
+                            </ul>
+                        </div>
+                        <div class="underline"></div>
+                        <div class="platform__wrapper">
+                            <h3 class="frameworks-title">Skills</h3>
+                            <ul class="bar__wrapper">
+                                <li class="bar"></li>
+                                <li class="bar"></li>
+                            </ul>
+                        </div>
+                        <div class="underline"></div>
+                    </div>         
+                </div>
+            </div>
+            
+        </div>
+        
+    </section>
+
+    <section class="contact-me_page">
+        <div class="contact__wrapper">
+            <h2 class="contact__header">Contact me</h2>
+            <p class="description__contact">I am a software developer! I can help you build a product, feature or website look through some of my work and experiance! if you like what you see or have a project you need coded don't hesitate to contact me. </p>
+            <form>
```

Comment:
Please add form validations, `action`, and `post` attributes. Moreover please use `Formspree` to submit the message to user's email.